test/execute_tests_utest.py

```diff
@@ -140,7 +140,7 @@ def test_with_aspect(self):
         cmd = make_cmd(test_cmd=TestCmd(target="//foo:bar"), extra_args=[])
         self.assertEqual(cmd, self._base_cmd() + ["--", "//foo:bar"])
 
-    def test_with_aspect(self):
+    def test_with_aspect_2(self):
```

Comment:
I don't understand why there are two functions with same name @martis42

Reply:
Copy & Paste error on my side. Will add a commit to fix this.